owncloudApp/src/main/java/com/owncloud/android/ui/activity/Preferences.java

```diff
@@ -513,15 +507,36 @@ public void onCreate(Bundle savedInstanceState) {
             ));
             mAboutApp.setSummary(String.format(getString(R.string.about_version), appVersion));
             mAboutApp.setOnPreferenceClickListener(preference -> {
-                String commitUrl = BuildConfig.GIT_REMOTE + "/commit/" + BuildConfig.COMMIT_SHA1;
-                Uri uriUrl = Uri.parse(commitUrl);
-                Intent intent = new Intent(Intent.ACTION_VIEW, uriUrl);
-                startActivity(intent);
+                int clickCount = mAppPrefs.getInt(MainApp.CLICK_DEV_MENU, 0);
+                if (mAppPrefs.getInt(MainApp.CLICK_DEV_MENU, 0) > MainApp.CLICKS_NEEDED_TO_BE_DEVELOPER) {
+                    String commitUrl = BuildConfig.GIT_REMOTE + "/commit/" + BuildConfig.COMMIT_SHA1;
+                    Uri uriUrl = Uri.parse(commitUrl);
+                    Intent intent = new Intent(Intent.ACTION_VIEW, uriUrl);
+                    startActivity(intent);
+                } else if (mAppPrefs.getInt(MainApp.CLICK_DEV_MENU, 0) == MainApp.CLICKS_NEEDED_TO_BE_DEVELOPER) {
+                    showDeveloperItems(pCategoryMore);
+                } else if (clickCount > 0) {
+                    Toast.makeText(this,
+                            getString(R.string.clicks_to_be_developer, MainApp.CLICKS_NEEDED_TO_BE_DEVELOPER - clickCount),
+                            Toast.LENGTH_SHORT).show();
+                }
+                mAppPrefs.edit().putInt(MainApp.CLICK_DEV_MENU, clickCount + 1).apply();
+                ((MainApp) getApplication()).readIsDeveloper(); // read value to global variable
                 return true;
             });
         }
     }
 
+    private void showDeveloperItems(PreferenceCategory preferenceCategory) {
+
```

Comment:
Delete this line

Reply:
When I would search code for that, I expect a lot of this. But maybe I'm wrong
Done